File: 3154-maximum-value-of-an-ordered-triplet-i/maximum-value-of-an-ordered-triplet-i.java
class Solution {
    public long maximumTripletValue(int[] nums) {
        long maxi=0;
        for(int i=0;i<nums.length;i++)
        {
            for(int j=i+1;j<nums.length;j++)
            {
                for(int k=j+1;k<nums.length;k++)
                {
                    maxi=Math.max(maxi,(long)(nums[i]-nums[j])*nums[k]);
                }
            }
        }
        return maxi;
    }
}

File: 3154-maximum-value-of-an-ordered-triplet-i/maximum-value-of-an-ordered-triplet-i.py
class Solution:
    def maximumTripletValue(self, nums: List[int]) -> int:
        maxi=0
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                for k in range(j+1,len(nums)):
                    maxi=max(maxi,(nums[i]-nums[j])*nums[k])
        return maxi